docs/source/authoring_flows.rst

```diff
@@ -519,17 +519,190 @@ Example Flows
 "Move (copy and delete) files using Globus"
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Flow ID: ``f37e5766-7b3c-4c02-92ee-e6aacd8f4cb8``.
-
 Perform a 'move' operation on a directory by first transferring
 from a source to a destination and then deleting the directory from the source.
 The entire directory's contents, including files and subdirectories, will be
 copied to the destination and then deleted from the source.
 
-Note that this flow requires at least one of the collections to be managed under a Globus subscription.
+.. tip::
+   To see a complete example that includes error handling, view the `Move (copy and delete) files using Globus`_ in the Globus web app (Flow ID: ``f37e5766-7b3c-4c02-92ee-e6aacd8f4cb8``).
+
+   Note: This flow requires at least one of the collections to be managed under a Globus subscription.
 
-View the `Move (copy and delete) files using Globus`_ in the Globus web app.
-(You may need to log in first.)
+.. code-block:: json
+    :caption: Example Definition
+
+    {
+        "States": {
+            "LookupSourcePath": {
+                "Next": "SetSourceInfo",
```

Comment:
I kinda think `Next` and `End` tags should go at the end of the state definition rather than the start. My line of thinking would be
1. The reader should learn what the thing is from the first couple attributes attributes (type, actionurl, and comment should adequately cover that)
2. Putting next in the definition will have a direct link (because of the sequential state authoring you did here) to the state 2 lines below meaning the reader isn't ever required to jump around the file